### DISCUSSION TITLE
Inventory on hand/Inventory Checklist

### DISCUSSION THREAD

**Discussion by fdimatt** [blueseerERP/blueseer#112] [Q&A] [answered]
Hi.

I am new to BlueseerERP.  I was wondering if I am able to run an inventory on hand or inventory checklist that shows all stocked inventory.  I am hoping to create a report/checklist that can be used for 1). conducting a physical inventory and 2) showing what is physically on hand and its value at any given time.

Much Appreciated!
Frank

**Comment by vaughnte** [comment, marked as answer]
There are two reports that show Quantity on Hand (QOH).   The item browse menu...under the Inventory->ItemMenu and the 'Inventory Report Selector' under the Inventory Menu.   The navcodes for both of these are 'itemb' and 'rpin' respectively.   You can type either of these two codes in the navcode text-box on the main menu bar and it will take you to the respective menus.   BlueSeer does not have an inventory 'value-at' by date....only current value of QOH times Cost.
